modules/ping-protection/models/ModerationLog.js

```diff
@@ -0,0 +1,39 @@
+const { DataTypes, Model } = require('sequelize');
+
+module.exports = class ModerationLog extends Model {
```

Comment:
```suggestion
module.exports = class PingProtectionModerationLog extends Model {
```
ensure module name is contained in class name to avoid sequelize issues, also name is not OOP